python/lsst/ctrl/execute/slurmPlugin.py

```diff
@@ -100,14 +102,9 @@ def countRunningSlurmJobs(jobname):
         numberOfJobs = SlurmPlugin.countSlurmJobs(jobname, jobstates="R")
         return numberOfJobs
 
-    def createFilesFromTemplates(self, platformPkgDir):
+    def createFilesFromTemplates(self):
```

Comment:
Is this an internal API or a public API that is changing here and below? I don't know enough about how this is called to know if there is some kind of backwards compatibility issue here. Even if it's keeping `platformPkgDir: str = ""` around in the API for a short while and warning if it's set to anything.

Reply:
Looks like an internal API to me (it's not defined on the parent class at all) and is not called by any code in either `lsst` or `lsst-dm`.